Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -4894,6 +4916,14 @@ - (void)reloadViewItems
         }
     }];
 
+    // Flag to ensure that we only increment once per launch.
+    static BOOL hasIncrementedDatabaseView = NO;
+    if (hasError && !hasIncrementedDatabaseView) {
+        DDLogWarn(@"%@ incrementing version of: %@", self.logTag, TSMessageDatabaseViewExtensionName);
+        [OWSPrimaryStorage incrementVersionOfDatabaseExtension:TSMessageDatabaseViewExtensionName];
+        hasIncrementedDatabaseView = YES;
```

Comment:
This class of error indicates that we need to rebuild our database view.  I've confirmed through Riya that this resolves the issue.